frontend/javascripts/oxalis/view/settings/user_settings_view.js

```diff
@@ -425,62 +425,64 @@ class UserSettingsView extends PureComponent<UserSettingsViewProps> {
     );
 
     return (
-      <React.Fragment>
-        <Collapse bordered={false} defaultActiveKey={["1", "2", "3a", "3b", "4"]}>
-          <Panel header="Controls" key="1">
-            <NumberSliderSetting
-              label={settingsLabels.keyboardDelay}
-              min={userSettings.keyboardDelay.minimum}
-              max={userSettings.keyboardDelay.maximum}
-              value={this.props.userConfiguration.keyboardDelay}
-              onChange={this.onChangeUser.keyboardDelay}
-            />
-            {moveValueSetting}
-            <SwitchSetting
-              label={settingsLabels.dynamicSpaceDirection}
-              value={this.props.userConfiguration.dynamicSpaceDirection}
-              onChange={this.onChangeUser.dynamicSpaceDirection}
+      <Collapse
```

Comment:
The following diff is messed up. As far as I can remember I only removed the wrapping `React.Fragment`. The Fragment was unnecessary here.